### PR TITLE
feat: Add simple footer

### PR DESCRIPTION
**Pull request by micgro42**
The main purpose of this footer is to show the last build time, so we
can have an idea when the production environment is updated again.

Proper (translatable) copy will be implemented later.

Bug: [T268643](https://phabricator.wikimedia.org/T268643)

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<template>
+	<footer class="querybuilder-footer">
+		<p>
+			Made by the Wikimedia Deutschland Wikidata team.
+			Licensed under <a href="https://github.com/wmde/query-builder/blob/master/LICENSE">BSD 3-Clause License</a>.
+			View on <a href="https://github.com/wmde/query-builder">GitHub</a>.
```
Comment (Ladsgroup) [inline src/components/Footer.vue]: "View source" is a better word IMO.

Comment (micgro42) [reply src/components/Footer.vue]: Will adjust

Comment (micgro42) [reply src/components/Footer.vue]: Done

**Comment by guergana** [comment]
I dont know what the current discussion about this is, but it looks good.

**Review by guergana — APPROVED** [review]
Looks good. I haven't participated in the discussions around the footer but looks great.

**Pull request merged**